client.go

```diff
@@ -140,6 +140,21 @@ func injectClient(a *auth) *Client {
 	return c
 }
 
+func (c *Client) executeRaw(method string, urlStr string, text string) ([]byte, error) {
```

Comment:
When requesting file content from Bitbucket, a blob is returned; however, the existing `execute` routine expected a JSON payload to be returned.  As such, `execute` in its prior state resulted in errors due to an attempt to parse the response.  This function was introduced so we could get the raw payload back from the Bitbucket API and pass that along to the consumer.

Reply:
That's right, and I understood it. LGTM! :+1: